crates/ruff_python_parser/src/parser/expression.rs

```diff
@@ -1225,6 +1243,28 @@ impl<'src> Parser<'src> {
             }
         }
 
```

Comment:
Github doesn't allow me to comment on the relevant line: 

```rust
                    for string in strings {
                        values.push(match string {
                            StringType::Bytes(value) => value,
                            _ => ast::BytesLiteral::invalid(string.range()),
                        });
                    }
```

For the above code, isn't it impossible to have an element that isn't `StringType::Bytes`, given that `bytes_literal_count == strings.len()`?